### ISSUE TITLE
Feature Request - Add an option to add the default language comments to all language files

### ISSUE THREAD

**Issue by bhartsfield** [tom-englert/ResXResourceManager#81]
We need to have the default language comments (English in our case) carried into ALL of the resulting language files. I am not seeing a way to do it nor do I see any options around it.

**Comment by tom-englert** [comment]
You could use the [scripting](../wiki/Scripting) module to synchronize the comments

**Comment by tom-englert** [comment]
@bhartsfield any luck trying to script this?

**Comment by bhartsfield** [comment]
No. I haven't had the time to look into it. The person managing the files is just doing it manually until I can put some time into it.
